src/core/linux/X11.zig

```diff
@@ -65,9 +65,6 @@ hidden_cursor: c.Cursor,
 // Mutable fields only used by main thread
 cursors: [@typeInfo(CursorShape).@"enum".fields.len]?c.Cursor,
 
-// Input state; written from main thread; read from any
-input_mu: std.Thread.RwLock = .{},
```

Comment:
I think that the removal of this mutex is probably wrong; however I expect the external API for input handling to change greatly so totally fine for now.